static/_redirects

```diff
@@ -445,25 +444,24 @@
 /horizontal-pod-autoscaler/     /docs/tasks/run-application/horizontal-pod-autoscale/ 301
 /kubernetes/     /docs/ 301
 /kubernetes-bootcamp/*     /docs/tutorials/kubernetes-basics/ 301
-/kubernetes/swagger-spec/     https://github.com/kubernetes/kubernetes/tree/master/api/swagger-spec/ 301
+# /kubernetes/swagger-spec/     https://github.com/kubernetes/kubernetes/tree/master/api/swagger-spec/ 301
 /kubernetes/third_party/swagger-ui/     /docs/reference/ 301
 /latest/docs/     /docs/home/ 301
 /media/     /docs/community     301
 /news/     /docs/community     301
 /resource-quota/     /docs/concepts/policy/resource-quotas/ 301
 /serviceaccount/token/     /docs/tasks/configure-pod-container/configure-service-account/ 301
-/swagger-spec/*     https://github.com/kubernetes/kubernetes/tree/master/api/swagger-spec/ 301
+# /swagger-spec/*     https://github.com/kubernetes/kubernetes/tree/master/api/swagger-spec/ 301
```

Comment:
I'd consider leaving this in: if someone is following that link, at least the GitHub 404 gives you a clue about where the content once lived.
(could also file an issue for someone to fix the redirect)

Reply:
This is true, though, why should the site provide a `/swagger-spec/*`?

Reply:
I'm assuming that there were once inbound links there. Not a big deal; removal is also fine.

Reply:
I'm cleaning, unless there are objections :smile: